content/docs/reference/pkg/nodejs/pulumi/kubernetes/apps/v1beta1/_index.md

```diff
@@ -11,150 +11,17 @@ linktitle: "v1beta1"
 
 <h3>Resources</h3>
 <ul class="api">
-    <li><a href="#ControllerRevision"><span class="symbol resource"></span>ControllerRevision</a></li>
     <li><a href="#ControllerRevisionList"><span class="symbol resource"></span>ControllerRevisionList</a></li>
-    <li><a href="#Deployment"><span class="symbol resource"></span>Deployment</a></li>
     <li><a href="#DeploymentList"><span class="symbol resource"></span>DeploymentList</a></li>
-    <li><a href="#StatefulSet"><span class="symbol resource"></span>StatefulSet</a></li>
     <li><a href="#StatefulSetList"><span class="symbol resource"></span>StatefulSetList</a></li>
 </ul>
 
 
 
 
 <h2 id="resources">Resources</h2>
-<h3 class="pdoc-module-header" id="ControllerRevision" data-link-title="ControllerRevision">
```

Comment:
@justinvp Do you know if it would have anything to do with setting the `@deprecated` property in the SDK?

Reply:
> Do you know if it would have anything to do with setting the `@deprecated` property in the SDK docs?

Yes, `tsdocgen` [strips APIs tagged as `@deprecated`](https://github.com/pulumi/docs/pull/1469) in the generated docs.

Reply:
Opened #1766 to track not filtering out @deprecated APIs